geometry_msgs/PoseStamped pose_stamped
sensor_msgs/Image image
ImagePoint[] blob_cogs
ProjectedPoint[] blob_proj
